### ISSUE TITLE
Hamburger menu ?

### ISSUE THREAD

**Issue by gedw99** [golang-cafe/job-board#100]
The current tab like menu is ok .

a hamburger menu would be easier because it will work better on mobile but still be fine on desktop.

This one works just fine and has example and would be easy to integrate 👍 

https://www.w3schools.com/howto/howto_js_mobile_navbar.asp

oh god it’s an asp backend !!

**Comment by dm161** [comment]
@gedw99 I agree i don't have particular front-end skills myself so there are several items to be improved on mobile ui/ux. For instance also the signup popup banners are not mobile friendly. as also the ui is a bit crammed and difficult to navigate. I would like to improve the site and make it really good on mobile. At the moment I am trying to make the site grows so i can invest more on all of this. I am not sure if improving mobile right now will bring more customers or engagement, but I might be wrong. I am all in for this, but I dont have the budget right now to prioritise it. If the platform grows substancially I will have budget in the near future (2-3 months).

If you want to open PR please feel free to do so. I will review this soon

**Comment by gedw99** [comment]
Mobile is 80% on average. Your analytics will show the real number.

the site works quite well on mobile actually !! 

Only things I see are:

- nav cleanup
- Font size is a little too big . On both D and M so easy to fix.
- feedback in lower right is just weird with many issus. Better as a dialogue or a slight out from the right. 
- that’s about it for the first pass and willl get 80% improvement for 20% effort. Know what I mean I think: 80/20 rule

**Comment by dm161** [comment]
Desktop users represents 90% of the traffic at the moment, so mobile UI/UX is not currently a priority but these are definitely items we can work on, just with lower priority. I can create a bunch of issues so people can work on. Thanks for the feedback!
